ci/docker-compose.prod.yaml

```diff
@@ -37,8 +37,44 @@ services:
     image: gcr.io/google.com/gchips-productivity/resultstoresearch-server
     environment: 
       GOOGLE_APPLICATION_CREDENTIALS: /root/keys/keyfile.json
+      DATABASE_HOST: db
+      DATABASE_USERNAME: postgres
+      DATABASE_PASSWORD: postgress
+      DATABASE_PORT: 5432
+      DATABASE_NAME: resultstore_search
     volumes:
       - ../resultstore-search-api/:/usr/src/app
       - ~/service_account.json:/root/keys/keyfile.json:ro
     ports:
       - "9090:9090"
+    depends_on:
+      - db
+
+  db:
+    build:
+      context: ../db
+    environment:
+      - POSTGRES_USER=postgres
```

Comment:
Do we have to use a sql db? If not, we can use cloud datastore. It's super easy and 0 setup.

Reply:
Talked to @J-Pai about this yesterday, we came to the conclusion to use a self hosted db, just so the project isn't fully tied to gcp in-case customers want to deploy else-where. But if you want to use cloud datastore instead that's fine.

Reply:
we are using a lot of google services so whoever wants to deploy it must have a GCP instance anyways. enabling cloud datastore is minimal work. schema/entities are automatically generated when data are piped in. So it should be highly movable to other gcp instances too.

Reply:
The way the project is currently setup, the users don't necessary need to run this off a GCP instance. The project can also be setup to run locally. Providing the option to use a different database for this work can help on costs and improve setup time.

Instead of having the users needing to setup multiple dependencies in GCP, the user can just execute the docker-compose files and have everything setup ready to go.